src/commcare_cloud/ansible/roles/kafka/templates/server.properties.j2

```diff
@@ -19,6 +19,13 @@
 # The id of the broker. This must be set to a unique integer for each broker.
 broker.id={{ kafka_broker_id }}
 
+{% if wipe_environment_enabled|default(False) %}
+################################## Bad Ideas ##################################
+
+# Allow topics to be deleted
+delete.topic.enable=true
```

Comment:
I was thinking about this and instead of putting this in the config you could use the `lineinfile` module to add / remove it which would be much easier than running the whole kafka role

Reply:
Thank you! I'll do that.